scipy/special/__init__.py

```diff
@@ -817,3 +822,8 @@
 from scipy._lib._testutils import PytestTester
 test = PytestTester(__name__)
 del PytestTester
+
+btdtr = np.deprecate(btdtr, "btdtr", "betainc")
+btdtri = np.deprecate(btdtri, "btdtri", "betaincinv")
```

Comment:
We should make sure that these warnings state when they will be removed

Reply:
I added information about SciPy versions that will show up in the docstring and in the deprecation warning.